Sim/SimPapas/SimPapas/IPapasTool.h

```diff
@@ -3,8 +3,14 @@
 
 // Gaudi
 #include "GaudiKernel/IAlgTool.h"
+// Papas
+#define WITHSORT 1
```

Comment:
Can this WITHSORT be replaced by a static variable in the papas namespace? 
There is a risk of collision if another WITHSORT is defined somewhere else by somebody else